doc/tutorials/klondike/step5.md

```diff
@@ -0,0 +1,604 @@
+# 5. Additional features
+
+In this chapter we will be showing various ways to make the Klondike game more fun and easier to
+play. As it is now, the cards move from one pile to another immediately, which looks unnatural, so
+we will also be working on ways to animate that and smooth it out, for which we will be introducing
+Effect and EffectController components.
+
+
+## The Klondike draw
+
+The Klondike patience game (or solitaire game in the USA) has two main variants: Draw 3 and Draw 1.
+Currently the Klondike Flame Game is Draw 3, which is a lot more difficult than Draw 1, because
+although you can see 3 cards, you can only move one of them and that move changes the "phase" of
+other cards. So different cards are going to become available &mdash; not easy.
```

Comment:
```suggestion
other cards. So different cards are going to become available &mdash; not easy.
```
I'm not sure `&mdash;` will render properly in all markdown renderers, better use the unicode symbol for it.

Reply:
Hmmm. I used `&mdash;` because I read somewhere that it is the official HTML 5 code for that symbol. Surely any Markdown renderer would merely pass that through to its HTML 5 output... I could use `&#x2014;` throughout, but that is somewhat unreadable for anyone who needs to edit `step5.md` in the future. 2014 is the hex Unicode for mdash.

Alternatively, I could  use parentheses or some other symbol wherever mdash occurs. I like mdash because it maintains the conversational tone of the Klondike Tutorial.

Reply:
You don't have to write the hexcode for the unicode symbol, you can just copy the unicode symbol directly in there, like —. Then it is also easier to read when it is not rendered markdown.

Reply:
I see, you just copy and paste the glyph, or any glyph, from an authoritative Unicode website. My text-editor accepts and displays that OK too, though I suppose it is conveyed as a string of utf8 characters. Wow, that's easy,

Reply:
I have now replaced all the &mdash; strings with Unicode glyphs or alternative punctuation.